exercises/robot-simulator/src/lib.rs

```diff
@@ -0,0 +1,32 @@
+#[derive(PartialEq, Debug)]
+pub enum Direction {
+    North,
+    East,
+    South,
+    West,
+}
+
+pub struct Robot {
```

Comment:
I would personally use `pub struct Robot;`, because it feels a bit less pushed towards "normal" structs, because a tuple struct like `Robot(Direction, i32, i32)` would also be possible.